examples/Rakefile

```diff
@@ -1,41 +1,111 @@
 require 'fastly_nsq'
 require 'fastly_nsq/rake_task'
```

Comment:
You commented that one of these is not needed? We should remove it from here as well.